src/h5web/App.tsx

```diff
@@ -9,9 +9,15 @@ import styles from './App.module.css';
 import { isDataset } from './providers/utils';
 import { useEntity } from './providers/hooks';
 
+enum Role {
+  Inspect,
+  Visualize,
+}
```

Comment:
I found clearer to use an `enum` here rather than a simple boolean.

Reply:
I was wondering about this. It's quite clear and scalable with an enum, but since it's in App, I feel like the shorter the implementation, the better. Also, the fact that the display logic in the JSX checks only one of the roles and doesn't care about the other one makes me think that a boolean would be more appropriate.

That being said, if you feel strongly about an enum, I don't mind it.

Reply:
I agree for the display logic but it makes more sense for the rest. I tried first with a boolean but it felt awkward.